src/cgt/include/cgt/ClockGating.h

```diff
@@ -0,0 +1,197 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright (c) 2025, The OpenROAD Authors
+
+#pragma once
+
+#include <filesystem>
+#include <unordered_set>
+
+#include "cgt/NetworkBuilder.h"
+#include "cgt/RandomBits.h"
+#include "db_sta/dbSta.hh"
+#include "utl/deleter.h"
+
+namespace abc {
+using Abc_Ntk_t = struct Abc_Ntk_t_;
+using Abc_Obj_t = struct Abc_Obj_t_;
+using Vec_Ptr_t = struct Vec_Ptr_t_;
+}  // namespace abc
+
+namespace utl {
+class Logger;
+}
+
+namespace odb {
+class dbDatabase;
+}  // namespace odb
+
+namespace sta {
+class dbSta;
+}  // namespace sta
+
+namespace lext {
+
+class AbcLibrary;
+class AbcLibraryFactory;
+
+}  // namespace lext
+
+namespace cgt {
+
+using utl::Logger;
+
+enum class GroupInstances
+{
+  No,
```

Comment:
What is 'No'?  Is that no grouping?  If so I would use None.  

Using https://google.github.io/styleguide/cppguide.html#Enumerator_Names (leading k).